test/SdkE2ETests/InnerBuildTests.cs

```diff
@@ -47,7 +42,12 @@ public async Task Build_ScansReferences()
       ""name"": ""AzureStorageQueues"",
       ""typeName"": ""Microsoft.Azure.WebJobs.Extensions.Storage.AzureStorageQueuesWebJobsStartup, Microsoft.Azure.WebJobs.Extensions.Storage.Queues, Version=5.1.3.0, Culture=neutral, PublicKeyToken=92742159e12e44c8"",
       ""hintPath"": ""./.azurefunctions/Microsoft.Azure.WebJobs.Extensions.Storage.Queues.dll""
-    }
+    },
+    {
+      ""name"": ""Startup"",
+      ""typeName"": ""Microsoft.Azure.WebJobs.Extensions.FunctionMetadataLoader.Startup, Microsoft.Azure.WebJobs.Extensions.FunctionMetadataLoader, Version=1.0.0.0, Culture=neutral, PublicKeyToken=551316b6919f366c"",
+      ""hintPath"": ""./.azurefunctions/Microsoft.Azure.WebJobs.Extensions.FunctionMetadataLoader.dll""
+    },
```

Comment:
Does the order matter here?

Reply:
Yes. The change made in this PR to manually scan via `AdditionalExtensions` happens _after_ the current extension scanning process. This test change reflects how that metadata loader startup extension is now part of the 'additional extensions' process.